infrastructure/src/main/java/org/corfudb/infrastructure/log/FileSystemAgent.java

```diff
@@ -246,5 +250,12 @@ public static class PartitionAttribute {
             private final long availableSpace;
             private final long totalSpace;
         }
+
+        /**
+         * Clean up
+         */
+        public static void shutdown() {
+            scheduler.shutdown();
+        }
```

Comment:
The usage of static is a little strange here - for a shutdown method. If the executor is a static, then shouldn't shutdown cancel task instead of shutting down the executor?

Reply:
Fixed.